apps/spruce/src/pages/projectSettings/tabs/GeneralTab/transformers.ts

```diff
@@ -36,8 +36,7 @@ export const gqlToForm = ((data, options = {}) => {
           "identifier" in projectRef && {
             identifier: projectRef.identifier,
```

Comment:
I think you might want to make some adjustments to this PR because I can't see this placeholder anymore:

<img width="444" alt="Screenshot 2024-08-14 at 5 35 59 PM" src="https://github.com/user-attachments/assets/812017c5-e9f8-4e56-a30a-a53ecc897f6e">

Now it says the batch time is 0, even though it should be 5 since it inherited from repo